Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandard/README.md

```diff
@@ -3,7 +3,23 @@ This integration was integrated and tested with version 1.1.2 of Carbon Black En
 
 ## The changes v1
 The old integration is deprecated because Carbon Black have released a new version of their API.
-The new integration supports new commands, Also added a Mapper and a Layout.
+The new integration supports new commands:
```

Comment:
is there a difference in the old commands? did we change inputs/outputs?
did we stop supporting specific commands (if so, why?)